index.html

```diff
@@ -81,16 +29,12 @@
         <section class="bio">
             <h1 class="intro">Welcome to YAOTIAN TONG’s CSD120 Website</h1>
             <figure class ="mugshot">
-                <div>
-                <figure id="example1" >
                 <a href="stylesheet"><img src="stylesheet/me.jpg" alt="me" width="80" height="80" ></a>
-                </figure>
-                </div>
                 <figcaption>A picture of me</figcaption>
             </figure>
```

Comment:
You already have a `<figure>` element here.  No need to add the extra div and figure elements.